oligo_designer_toolsuite/IO/_database.py

```diff
@@ -383,9 +379,10 @@ def __init__(
 
         genome_assembly = "GRCh38"
```

Comment:
genome assembly should not be harcoded, since it depends on the species and annotation release and should be returned correctly

Reply:
I would modify this in the branch where we split the reference DB from the oligos DB. So w can also minimize the merge conflicts with the dev branch.

Reply:
ok, sounds good